### PR TITLE
bugfix #460: added correct sorting of teachers

### LINKED ISSUE #408: Create pagination for responses on teacher
Now, if you open tab with responses on teacher, there will be all responses that are stored in database. It's not good decision because there can be more than 50 responses on one teacher. The solution is to create pagination like on subjects and teachers pages

### LINKED ISSUE #460: Bug with sorting
When we open teachers page, we can see this sorting:

image

But when we pressed "download more" button, order changes:
opera_MDaaggXRUY

No any filters set: 
image

We need to sort appropriately after the first downloading

### PR DESCRIPTION
**Pull request by SharpDevOps10**
- added several sorting fields
- added array to which sort objects are attached for each field
- sorted all fields

closes #460

### CODE REVIEW

**Review by Hahhhayam — CHANGES_REQUESTED** [review]
remove unnessary varaibles, like: 
image
if you think, that your version more readable, close this request

**Review by Hahhhayam — APPROVED** [review]

**Review by f-ogoog — APPROVED** [review]

File: src/v2/mappers/TeacherMapper.ts [hunk 1]
```diff
@@ -26,6 +30,20 @@ export class TeacherMapper {
     return roles;
   }
 
+  getSortedTeacher ({ sort, order = OrderQAParam.ASC }: SortDTO): Sort | object {
```
Comment (just-hoshinon) [inline src/v2/mappers/TeacherMapper.ts:33]: you can do sort as you do orded:
```ts
{ sort = SortQATParam.LAST_NAME, order = OrderQAParam.ASC }
```

File: src/v2/mappers/TeacherMapper.ts [hunk 2]
```diff
@@ -26,6 +30,20 @@ export class TeacherMapper {
...
+    if (!sort) sort = SortQATParam.LAST_NAME;
```
Comment (just-hoshinon) [inline src/v2/mappers/TeacherMapper.ts:34]: this field will be redundant

File: src/v2/mappers/TeacherMapper.ts [hunk 3]
```diff
@@ -26,6 +30,20 @@ export class TeacherMapper {
...
+
+    const orderBy = sort.split(',').map((field) => ({
```
Comment (just-hoshinon) [inline src/v2/mappers/TeacherMapper.ts:36]: front-end will never send more than one value from enum, so splitting and mapping is reduntant, as well as trimming

**Review by just-hoshinon — CHANGES_REQUESTED** [review]

**Review by just-hoshinon — APPROVED** [review]

**Pull request merged**